pyproject.toml

```diff
@@ -118,37 +123,66 @@ default = {features = ["test", "package", "docs", "developer"], solve-group = "d
 
 [tool.pixi.pypi-dependencies]
 plot-publisher = { path = ".", editable = true }
+toml-cli = "*"  # Required for sync-version task
 
 [tool.pixi.dependencies]
 python = ">=3.9"
 numpy = "*"
 requests = "*"
 plotly = "*"
 
+# Pixi Build configuration for conda packaging
+[tool.pixi.package.host-dependencies]
+hatchling = "*"
+versioningit = "*"
+
+[tool.pixi.package.run-dependencies]
+python = ">=3.9"
+numpy = "*"
+requests = "*"
+plotly = "*"
+
+[tool.pixi.package]
+name = "plot-publisher"
+version = "0.0.0"  # placeholder, overwritten by sync-version
+
+[tool.pixi.package.build]
+backend = { name = "pixi-build-python", version = "0.1.*" }
+
 # --------------- #
 # Pixi Tasks      #
 # --------------- #
 [tool.pixi.tasks]
 # General
 lint = { cmd = "pre-commit run --all-files", description = "Lint the project" }
 test = { cmd = "pytest", description = "Run the tests" }
-# Build
-build = {depends-on = ["build-conda", "build-pypi"]}
-build-conda = { cmd = "pixi run build-pypi && rattler-build build --recipe recipe/recipe.yaml", description = "Build the conda package" }
+# Build - Python Project Template approach
+sync-version = 'version=$(python -m versioningit); toml set tool.pixi.package.version "$version" --toml-path pyproject.toml'
+backup-toml = "cp pyproject.toml pyproject.toml.bak"
+reset-toml = "cp pyproject.toml.bak pyproject.toml; rm pyproject.toml.bak"
+conda-build-command = "pixi build"
+conda-build = { description = "Build the conda package", depends-on = [
+  "backup-toml",
+  "sync-version",
+  "conda-build-command",
+  "reset-toml",
+] }
 build-pypi = { cmd = "hatch build", description = "Build the package for PyPI" }
+build = {depends-on = ["conda-build", "build-pypi"]}
 # Docs
 docs = {depends-on = ["build-docs"]}
 build-docs = { cmd = "sphinx-build -b html docs docs/_build", description = "Build the documentation" }
 serve-docs = { cmd = "sphinx-autobuild docs docs/_build", description = "Serve the documentation" }
 # Release
-release = {depends-on = ["build", "publish-pypi", "publish-conda"]}
-publish-pypi = { cmd = "twine upload dist/*", description = "Publish the package to PyPI" }
-publish-conda = { cmd = "pixi upload conda-upload", description = "Publish the package to anaconda.org" }
+conda-publish = { cmd = "anaconda upload *.conda", description = "Publish the package to anaconda.org", depends-on = ["conda-build"] }
+publish-pypi = { cmd = "twine upload dist/*", description = "Publish the package to PyPI", depends-on = ["build-pypi"] }
+release = {depends-on = ["build", "publish-pypi", "conda-publish"]}
```

Comment:
perhaps we can remove these to avoid confusion, since we have the release pipeline defined in GitHub actions and publishing depends on the tests passing